streams/voxel_stream_region_files.cpp

```diff
@@ -1141,6 +1141,7 @@ void VoxelStreamRegionFiles::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("convert_files", "new_settings"), &VoxelStreamRegionFiles::convert_files);
 
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "channel", PROPERTY_HINT_ENUM, VoxelBuffer::CHANNEL_ID_HINT_STRING), "set_channel", "get_channel");
```

Comment:
In the case of files, it's again a bit strange to have channel settings, especially a single one. That choice may happen earlier than that, in the generator. But maybe there is another intention or use case behind this?

Reply:
No intention in the files. Just fumbling around to make the user interface simpler. I just wanted to remove the smooth meshing variable, and have the terrain nodes detect the channel type and it turned into a huge project.